lib/api/profile/index.js

```diff
@@ -15,13 +15,60 @@ function configure (app, wares, profile) {
     // also support url-encoded content-type
     api.use(wares.bodyParser.urlencoded({ extended: true }));
 
-    // List settings available
+    // List profiles available
     api.get('/profile/', function(req, res) {
         profile.list(function (err, attribute) {
             return res.json(attribute);
         });
     });
 
+	// List current active record (in current state LAST record is current active)
```

Comment:
It would be nice to get all these changes in `lib/api/profile/index.js` in a separate/simple PR